sql_metadata/parser.py

```diff
@@ -562,6 +569,45 @@ def with_names(self) -> List[str]:
         self._with_names = with_names
         return self._with_names
 
+    @property
+    def with_queries(self) -> Dict[str, str]:
+        """
+        Returns with subqueries with names
```

Comment:
```suggestion
        Returns "WITH" subqueries with names
```

I believe it's easier to read then a sentence with two withs 😉